README.rst

```diff
@@ -168,27 +169,27 @@ For more information on the contribution process, please go `here <CONTRIBUTING.
 MAPIE has been developed through a collaboration between Quantmetry, Michelin, ENS Paris-Saclay,
 and with the financial support from Région Ile de France and Confiance.ai.
 
-|Quantmetry|_ |Michelin|_ |ENS|_ |Confiance.ai|_  |IledeFrance|_ 
+|Quantmetry| |Michelin| |ENS| |Confiance.ai| |IledeFrance|
```

Comment:
In the following example, why not change `width` with `height`, so that all the logos are the same size?

Reply:
done.